src/components/CommonUserModal.tsx

```diff
@@ -307,7 +347,7 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
         open={open}
         onClose={onClose}
         showFooter={false}
-        modalTitle={!isEditModal?t("LEARNERS.NEW_LEARNER"): t("LEARNERS.EDIT_LEARNER")}
+        modalTitle={!isEditModal? userType === "STUDENT"?t("LEARNERS.NEW_LEARNER"):userType === "TEACHER"?t("FACILITATORS.NEW_FACILITATOR"):t("TEAM_LEADERS.NEW_TEAM_LEADER"): userType === "STUDENT"?t("LEARNERS.EDIT_LEARNER"):userType === "TEACHER"?t("FACILITATORS.EDIT_FACILITATOR"):t("TEAM_LEADERS.EDIT_TEAM_LEADER")}
```

Comment:
@AkshataKatwal16 use app.constant file for constants and try to move this logic to top of the file

Reply:
done